index.html

```diff
@@ -9317,6 +9321,7 @@ <h5>Presentational Roles Conflict Resolution</h5>
 			<rdef>textbox</rdef>
 			<div class="role-description">
 				<p>A type of input that allows free-form text as its value.</p>
+				<p>If the <pref>aria-valuetext</pref> <a>attribute</a> is set it indicates the <a>value</a> of the textbox. If it is not set then the value is the child content of the textbox.</p>
```

Comment:
I don't think this should be permitted if the textbox role is on an html input element. It needs language to the effect:

>If the textbox element is a host language element that provides a value, such as an HTML input  element, authors must not specify aria-valuetext.
>If the textbox element is a host language element that provides a value, such as an HTML input  element, user agents MuST ignore aria-valuetext if it is specified.